CMakeLists.txt

```diff
@@ -33,8 +33,11 @@ set(ENABLE_WARNINGS_AS_ERRORS On CACHE BOOL "")
 option(ENABLE_EXAMPLES "Build Umpire examples" On)
 option(ENABLE_LOGGING "Build Umpire with Logging enabled" On)
 option(ENABLE_SLIC "Build Umpire with SLIC logging" Off)
-option(ENABLE_ASSERTS "Build Umpire with assert() enabled" On)
-set(ENABLE_GTEST_DEATH_TESTS ${ENABLE_ASSERTS} CACHE BOOL "")
+if (CMAKE_BUILD_TYPE MATCHES "Debug")
```

Comment:
I don't think option is right here, since this is actually created in BLT. I don't think we should try and set it automatically here, let's just use BLT's default option setting, and allow users to override it with `-D` on the CMake command line.

In short, just get rid of all these lines.